### ISSUE TITLE
Setup Flow Nitpicks

### ISSUE THREAD

**Issue by tneotia** [BlueBubblesApp/bluebubbles-server#527]
- [x] show accessibility as optional in widget

![image](https://user-images.githubusercontent.com/50850142/235530439-813c8cd7-9baa-467c-ba4e-690594ddc9f4.png)

- [x] let's move the existing tedious setup guide to a docs page, and update website with a much simpler guide. as such, this needs to change:

![image](https://user-images.githubusercontent.com/50850142/235530497-dc92a7f5-e0b2-419c-b5e3-4f16a4114333.png)

- [x] add a log for "done" so the user isn't stuck there

<img width="646" alt="image" src="https://user-images.githubusercontent.com/50850142/235530829-1752eadb-7dbe-4e29-b12f-b0829d44dc3b.png">

- [x] indicate papi setup is optional in big text

<img width="487" alt="image" src="https://user-images.githubusercontent.com/50850142/235530943-f31745cb-9d49-4f3d-ab78-6b10fd222fff.png">

- [x] if accessibility is fail, gray out this option

<img width="393" alt="image" src="https://user-images.githubusercontent.com/50850142/235531062-9b718791-b179-4032-b05e-1fb09d3884f1.png">
